components/navbar.vue

```diff
@@ -59,3 +122,9 @@ export default defineComponent({
   },
 })
 </script>
+
+<style>
+.dropdown:hover .dropdown-menu {
+  display: block;
+}
```

Comment:
can be done with tailwind I believe

Reply:
Yes, actually in the main button tag i used tailwind but putting this part in styles seemed more appropriate.